homework.md

```diff
@@ -0,0 +1,19 @@
+# Describe shared testing activities for desktop and mobile
```

Comment:
Great observations Sila! I like how you gave examples to explain your thoughts.
However I get that the part where you were supposed to discuss shared activities felt more like an introduction and it did touch upon platforms uniqueness.
So maybe adding some specific activities would help clarifying this.